### PR TITLE
Error messages login

### PR DESCRIPTION
**Pull request by mahadaahmed**

### CODE REVIEW

**Comment by mahadaahmed** [comment]
Jag tog bort useState från view och la det i presenter istället. Bättre ha allt där i presenter.

File: pot-bot-app/src/App.js
Comment (RobertFuruvald) [inline pot-bot-app/src/App.js:1]: has not rebased this file with main

File: pot-bot-app/src/firebaseModel.js
Comment (RobertFuruvald) [inline pot-bot-app/src/firebaseModel.js:1]: has not rebased with main

**Review by RobertFuruvald — CHANGES_REQUESTED** [review]
Does not look like you have rebased with main.
firebaseModel.js is not rebased for instances so this will make our app to stop working
Fix this and send a new pull request

**Pull request closed without merging**